src/nvim/option.c

```diff
@@ -6911,15 +7043,309 @@ int check_ff_value(char_u *p)
   return check_opt_strings(p, p_ff_values, false);
 }
 
+// Set the integer values corresponding to the string setting of 'vartabstop'.
+// "array" will be set, caller must free it if needed.
+int tabstop_set(char_u *var, long **array)
```

Comment:
bool

Reply:
Are you referring to the second arg here? It is an actual array, even though I see why comment can make one think otherwise.

Reply:
Return value is true/false.